src/bindings/python/tests/test_runtime/test_model.py

```diff
@@ -146,6 +146,28 @@ def test_get_result_index_invalid():
     assert model.get_result_index(invalid_output) == -1
 
 
+def test_get_result_index_with_result():
+    input_shape = PartialShape([1])
+    param = ops.parameter(input_shape, dtype=np.float32, name="data")
+    relu = ops.relu(param, name="relu")
+    model = Model(relu, [param], "TestModel")
+    assert len(model.outputs) == 1
+    assert model.get_result_index(model.get_results()[0]) == 0
+
+
+def test_get_result_index_invalid_with_result():
+    shape1 = PartialShape([1])
+    param1 = ops.parameter(shape1, dtype=np.float32, name="data1")
+    relu1 = ops.relu(param1, name="relu1")
+    model1 = Model(relu1, [param1], "TestModel1")
+    shape2 = PartialShape([4])
+    param2 = ops.parameter(shape2, dtype=np.float32, name="data2")
+    relu2 = ops.relu(param2, name="relu2")
+    invalid_result_node = Result(relu2.outputs()[0])
```

Comment:
It'd be better to create result via `ops.result `. Actually it uses `openvino.runtime.op.Result` as backend

Reply:
done